src/components/CreateProvider.svelte

```diff
@@ -24,9 +25,8 @@
       $user.providerName = providerNameToHuman(msaInfo.providerName);
       $user.isProvider = msaInfo.isProvider;
       isInProgress = false;
-      setTimeout(() => {
-        pageContent.dashboard();
-      }, 1500);
+      // TODO: make nav reactive so we don't have to do this
+      goto('/');
```

Comment:
This is a fine pattern to use goto I think, but you will want to use `goto(base + '/')` (Base from `import { base } from '$app/paths';`) so that this works with a non-root deployment.